### ISSUE TITLE
Fructose malabsorption

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6657]
**Preferred term label:**
Fructose malabsorption

**Synonyms**
Intestinal fructose intolerance

**Definition (free text, please give PubMed ID)**
 Incomplete intestinal absorption of fructose which might lead to abdominal complaints such as pain, flatulence and diarrhoea. 
PMID: 26883354

**Parent term (use hpo.jax.org/app)**
Impairment of fructose metabolism

**Comment by pnrobinson** [comment]
This is essentially a synonym of the existing term
Fructose intolerance HP:0005973
I will add a synonym and this definition.

Reduced intestinal absorption of fructose. Breath testing after ingestion of fructose has been widely adopted as a standard method of identifying fructose malabsorption and intolerance. A dose of 25 g of fructose dissolved in a 10% solution is generally accepted as the appropriate dose of fructose for clinical use of H2 and CH4 breath testing. High levels of hydrogen in exhaled breath subsequent hours indicate fructose intolerance. Presence of malabsorption and reproduction of symptoms during a breath test provides the best objective evidence and symptom correlation for fructose intolerance. PMID:24357350

**Comment by pnrobinson** [comment]
added
